.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "Dev",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": { "VARIANT": "18" }
+	},
+
+	"customizations": {
+		"vscode": {
+			"extensions": [
+				"dbaeumer.vscode-eslint",
+				"ms-vscode.powershell",
+				"eg2.vscode-npm-script",
+				"mikestead.dotenv",
+				"christian-kohler.npm-intellisense"
+			]
+		}
+	},
+	"forwardPorts": [1521],
```

Comment:
It would also be possible to do

```json
	"postCreateCommand": "echo 'Use npm run dev to start watching for index.js changes'",
```

This is a bit controversial though https://stackoverflow.com/questions/55976358/should-a-vscode-dev-container-run-npm-install-as-part-of-its-setup

Thoughts?

Reply:
Decided to not do this but rather add npm install as part of the instructions on how to run instead